data/mainnet/pools.js

```diff
@@ -3091,16 +3091,16 @@ module.exports = [
   {
     tradingApyFunction: {
       type: TRADING_APY_TYPES.LP,
-      params: [addresses.BSC.pancake_CAKE_BNB.Underlying],
+      params: [addresses.BSC.V2.pancake_CAKE_BNB.Underlying],
     },
     chain: CHAINS_ID.BSC_MAINNET,
     id: 'pancakeCAKEBNB',
     type: POOL_TYPES.INCENTIVE,
-    contractAddress: addresses.BSC.pancake_CAKE_BNB.NewPool,
-    collateralAddress: addresses.BSC.pancake_CAKE_BNB.NewVault,
+    contractAddress: addresses.BSC.V2.pancake_CAKE_BNB.NewPool,
+    collateralAddress: addresses.BSC.V2.pancake_CAKE_BNB.NewVault,
     rewardAPY: [],
     rewardAPR: null,
-    rewardTokens: [addresses.BSC.bFARM, addresses.BSC.ampliFARM],
+    rewardTokens: [addresses.BSC.V2.bFARM, addresses.BSC.ampliFARM],
```

Comment:
I suppose `addresses.BSC.V2.bFARM` should be `addresses.BSC.bFARM`. Could you replace all instances?